guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,279 @@
+
+This approach makes a potential trade-off in terms of privacy, because the children could access the backup material and recover the wallet. This is acceptable for Alice and Bob, since they have good relationships and trust their children. After all, this is the reason they have chosen to include them in the setup in the first place.
+
+But even if Christina and David would breach their parents’ trust and import the wallet, they would not be able to move any funds, because their inheritance keys are time locked. And since Alice and Bob access their safe regularly, they would notice that the bag with the backup material has been opened. They can then safely move the funds to a new wallet, for which they would implement more restrictive security measures.
+
+### Inheritance planning
+
+Alice and Bob keep a copy of their Last Will in the home safe. It references the bitcoin wallet and provides a lead to Edward, their lawyer. To ensure a smooth inheritance process, Alice and Bob also provide a sealed copy of the Will to Edward.
+
+## Balancing security and privacy
+
+The setup oulined above addresses some important aspects of wallet backups:
+
+1. **Tamper-evidence**: The tamper-evident bag in the home safe allows Alice and Bob to quickly check if anyone has attempted to access the backup materials
+1. **Separation of components**: By keeping the encrypted backup kit with Edward and the password in the bank deposit box, the Jones family ensures that no single party can access the funds without proper authorization
+1. **Limited information sharing**: Edward can't monitor the wallet transactions because he doesn't have the password
+1. **Redundancy**: If the home safe becomes inaccessible, Christina and David can still recover the wallet through Edward and the bank deposit box
+1. **Simplicity**: The streamlined process makes it easier for the Jones family to maintain and use the backup system when needed
+
+## Recovery scenarios
+
+Now that we have a good understanding of the Joneses' backup solution, let's run through each of the three recovery scenarios to see how they work.
+
+#### Self-recovery (Alice and Bob)
+
+If Alice or Bob need to recover the wallet themselves they can easily do so by using the backup materials from the home safe or their password manager.
+
+If either Bob or Alice need to recover the funds alone, without the help of one another, they have two options. They can either use their own key plus the shared ColdCard, which is stored in a safe location. Alternatively they can wait for the [recovery path]({{ '/guide/inheritance-wallet/wallet-creation/#enabling-the-recovery-path' | relative_url }}) to kick in after 6 months. 
+
+#### Self-sovereign inheritance (Christina & David)
+
+If anything happens to Alice and Bob, Christina and David can access the funds after six months using:
+
+- The PIN to the safe, stored in the shared family vault of the password manager
+- The backup materials from the home safe
```

Comment:
```suggestion
- The backup materials from the personal vault
```

Suggestion here is to keep the word vault consistent throughout the copy. Variations in the copy are "tamper evident bag", "Home safe", "backup materials".

Reply:
As mentioned in the previous comments, "safe" and "vault" refer to two different types of vaults. I made some text updates that helps clarify this a bit.